README.md

```diff
@@ -1,24 +1,31 @@
-[![Join the chat at https://gitter.im/zlsa/atc](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/zlsa/atc?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
could you replace this with a link to our slack app?  there may be a badge for that, somewhere, too.